app/views/podcast/BodyComponent.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <div class="podcast-body">
+    <div class="podcast-content" v-if="podcastsLoaded">
+      <div
+        class="podcast-item"
+        v-for="podcast in allPodcasts"
+      >
+        <div class="container" v-if="podcast.visible">
+          <div class="row">
+            <router-link :to="{ name: 'PodcastSingle', params: { handle: podcast.slug } }">
+              <div class="col-md-6 podcast-item__info">
+                <div class="podcast-content__date">
+                  {{ getUploadDate(podcast.uploadDate) }}
+                </div>
+                <div class="podcast-content__title">
+                  {{ podcast.name }}
```

Comment:
These fields need to be using the translated versions if they exist. Usually we do [the `utils.i18n` function](https://github.com/codecombat/codecombat/blob/d30d89717c54ff4faf34ff2676fbb6708891cd06/app/core/utils.coco.coffee#L417-L417) for that.